modules/lti/copy-translations.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Accidental add?

Reply:
Not sure what you mean by that? The file is necessary and new.

Reply:
I hadn't caught that it was used, nevermind :)